### PR TITLE
Gnumeric

### PR DESCRIPTION
**Pull request by ghost**
Latest Gnumeric if I did things right?

### CODE REVIEW

**Comment by pyhalov** [comment]
Please, preserve only one last commit in this branch

File: components/desktop/gnumeric/Makefile [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+CONFIGURE_BINDIR.64 = $(CONFIGURE_BINDIR.32)
+
+CONFIGURE_OPTIONS+=	--sysconfdir=/etc
+CONFIGURE_OPTIONS+=	--libexecdir=$(CONFIGURE_LIBDIR.$(BITS))
+CONFIGURE_OPTIONS+=	--without-perl
+
+CONFIGURE_ENV+= PYTHON=$(PYTHON)
+
+COMPONENT_INSTALL_ENV+= GCONF_DISABLE_MAKEFILE_SCHEMA_INSTALL=1
```
Comment (pyhalov) [inline components/desktop/gnumeric/Makefile]: You likely can remove this line, as gconf is not used by gnumeric.

**Comment by pyhalov** [comment]
It seems, gnumeric doesn't accept non-ascii input...

**Comment by ghost** [comment]
Not sure about the non-ascii input, I see some searches that comment on PEP-538 with python and readline. Some of the sheets, I tested were in non-english language with characters I do not normally see.

**Pull request merged**